cli/init.md

```diff
@@ -24,6 +25,10 @@ Verify that you are running at least Node.js version 8.11+ or greater and npm ve
 $ npm install -g @aws-amplify/cli
 ```
 
+**Note**: The use of this CLI version >= 1.0, might cause existing projects initialized using a previous Amplify CLI version (< 1.0) to no longer function when attempting to manage resources in the existing project, or have unexpected side effects. After updating the CLI to >= 1.0, the CLI would prompt you to automatically migrate your project, so that is it compatible with the new version of the CLI. [Read more]({%if jekyll.environment == 'production'%}{{site.amplify.docs_baseurl}}{%endif%}/cli/migrate). CLI version >=1.0 supports multiple enviornments anf team workflows. [Read more]({%if jekyll.environment == 'production'%}{{site.amplify.docs_baseurl}}{%endif%}/cli/multienv)
```

Comment:
spelling of "and" is "anf"

Reply:
Hmm why didn't this typo get picked up in the spell checker CI test?